src/lib.rs

```diff
@@ -112,12 +124,42 @@ impl<K: Hash + Eq, V: Weighable> Default for WeightCache<K, V> {
     }
 }
 
+#[cfg(feature = "metrics")]
+struct Metrics {
+    hits: GenericCounter<AtomicU64>,
```

Comment:
Is this a re-export of std AtomicU64, or its own thing?

Reply:
It's their wrapper around it providing some trait impl they need.

Reply:
ok, so the usual rust newtype boilerplate...